### DISCUSSION TITLE
First Person Camera jitter, especially during the start of sprinting

### DISCUSSION THREAD

**Discussion by xkamronx** [Sixze/ALS-Refactored#355] [Q&A]
Camera doesn't move very smoothly in first person at times, especially when you start sprinting and try to look left and right slowly, extremely jittery / jumpy.

**Comment by Sixze** [comment]
This is by design. The camera is attached directly to the head bone and moves with it, so if there is jitter in the animation, the camera will also jitter.

**Reply by xkamronx** [reply]
I'm using the default animations, and it just feels very hard to play. Most beta testers have mentioned the same thing about first person just not feeling great.

Any tips for changes I can make to the camera to help with this at all?

**Reply by Sixze** [reply]
You need something like this: https://www.youtube.com/watch?v=rvwYASQsDMU

**Reply by xkamronx** [reply]
My main issue is turning left and right, it just gets jittery / not smooth at all. There's nothing we can do with the settings already available to help clean that up?

**Reply by xkamronx** [reply]
Doesn't seem like that video actually explains how to do it, and can't find anything in the marketplace at the moment that does the same thing. Will keep looking.
